src/ngraph/op/lrn.cpp

```diff
@@ -43,6 +44,15 @@ op::LRN::LRN(const Output<Node>& arg,
     constructor_validate_and_infer_types();
 }
 
+AxisSet op::LRN::get_reduction_axes() const
+{
+    auto axes_node = input(1).get_source_output().get_node_shared_ptr();
+    auto axes_constant = dynamic_pointer_cast<op::Constant>(axes_node);
+    auto axes_vector = axes_constant->get_vector<size_t>();
```

Comment:
I think there is some method like `get_axis_set_value()` you could use here.

Reply:
Done. It is `get_axis_set_val` ;)